SOPT-iOS/Projects/Features/HomeFeature/Sources/HomeScene/Cells/DefaultPost/DefaultPostCVC.swift

```diff
@@ -133,69 +143,81 @@ extension PlaygroundNewsCardCVC {
+        shapeLayer.lineWidth = borderWidth
+        shapeLayer.fillColor = UIColor.clear.cgColor
+        shapeLayer.strokeColor = UIColor.black.cgColor
+        shapeLayer.frame = bounds
+
+        // 그라데이션 색상 설정
+        gradientLayer.colors = [
+            DSKitAsset.Colors.orange300.color.cgColor,
+            DSKitAsset.Colors.orange200.color.cgColor,
+            DSKitAsset.Colors.orange500.color.cgColor
+        ]
+        
+        gradientLayer.startPoint = CGPoint(x: 0, y: 0.5)
+        gradientLayer.endPoint = CGPoint(x: 1, y: 0.5)
+        gradientLayer.locations = [0.0, 0.5, 1.0]
+        gradientLayer.mask = shapeLayer
+        gradientLayer.opacity = 0
+
+        if gradientLayer.superlayer == nil {
+            layer.addSublayer(gradientLayer)
+        }
+    }
 }
 
 // MARK: - Methods
 
-extension PlaygroundNewsCardCVC {
-    func configureCell(model: HomePresentationModel.PlaygroundNews) {
-        self.categorySubPhraseView.setData(with: "활동기수의 따끈한 새소식")
+extension DefaultPostCVC {
+    func configureCell(model: some PostDisplayable) {
+        self.categorySubPhraseView.setData(with: model.title)
         self.categoryTagView.setData(with: model.category)
-        self.userNameLabel.text = "김차돌"
-        self.userPartLabel.text = "32기 기획"
+        self.userNameLabel.text = model.name
+        self.userPartLabel.text = model.name
         if let profileImage = model.profileImage {
             self.profileImageView.setImage(with: profileImage)
         }
-        self.postTitleLabel.text = "나 메이커스팀인데 메팀 좋다"
-        self.postContentLabel.text = "본문 내용은 두줄로 보여줍니다. 본문 내용은 두줄로 보여줍니다.본문 내용은 두줄로 보여줍니다."
+        self.postTitleLabel.text = model.title
+        self.postContentLabel.text = model.content
         self.postContentLabel.setLineSpacing(lineSpacing: 1)
     }
 }
 
 
 // MARK: - Animation Methods
 
-extension PlaygroundNewsCardCVC {
-    func setOutlinedAnimated() {
-        self.layer.borderColor = DSKitAsset.Colors.orange300.color.cgColor
-        self.layer.borderWidth = 0
-        UIView.transition(with: self, duration: 1, options: .transitionCrossDissolve, animations: { [weak self] in
-            self?.changeComponentsUI(by: .focusing)
-        }) { _ in 
-            UIView.transition(with: self, duration: 1, options: .transitionCrossDissolve, animations: { [weak self] in
-                self?.changeComponentsUI(by: .unfocusing)
-            })
-        }
-    }
-    
-    private func animateColorFades(_ changes: [(UIView, () -> Void)]) {
-        for (view, change) in changes {
-            UIView.transition(with: view, duration: 0.3, options: .transitionCrossDissolve, animations: change, completion: nil)
-        }
+extension DefaultPostCVC {
+    /// 0.3초간 show -> 2.4초 기다림 -> 0.3초간 hide
+    func setOutlinedAnimated() async {
+        let interval: UInt64 = 2_400_000_000
+        
+        await animateBorderOpacity(to: 1)
+        try? await Task.sleep(nanoseconds: interval)
```

Comment:
```suggestion
        try? await Task.sleep(for: .seconds(2.4))
```
.seconds()로 표기하면 가독성 높일 수 있을 것 같아요~

Reply:
헉!! nanoseconds밖에 안되는 줄 알았어요